docs/release-notes/NuGet-6.10.md

```diff
@@ -191,4 +191,17 @@ Thank you to all the contributors who helped make this NuGet release awesome!
   * [5673](https://github.com/NuGet/NuGet.Client/pull/5673) Enable publishing in VMR
   * [5625](https://github.com/NuGet/NuGet.Client/pull/5625) Eliminate System.CommandLine prebuilt package
 * [brianrob](https://github.com/NuGet/NuGet.Client/pull/5650)
-  * [5650](https://github.com/NuGet/NuGet.Client/pull/5650) Add Restore Instrumentation
\ No newline at end of file
+  * [5650](https://github.com/NuGet/NuGet.Client/pull/5650) Add Restore Instrumentation
+
+## Summary: What's New in 6.10.1
```

Comment:
I think the newest version should be at the top of the document, not the bottom, so that customers viewing the page see the latest changes first.

Reply:
agreed, I was gonna ask how folks felt